.github/workflows/ci.yml

```diff
@@ -106,12 +106,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        browser: [chrome, firefox, edge]
+        browser: [[chrome, chromium], [firefox, firefox], [edge, chromium]]
```

Comment:
[Webdriver browser, Playwright browser]! Since I couldn't leave the second item empty, I put it there! Let me know if you got another way! Let me test webkit maybe.

Reply:
I don't know how CI works, but I would prefer it to be fixed _somehow_. I don't know _why_ you cannot provide an empty option.